ui/src/app/pages/profile/data-use-agreement-content.tsx

```diff
@@ -0,0 +1,200 @@
+      <li>My workbench and its contents may be made public for the benefit of all authorized users.</li>
+      <li>honor the contribution of those who take part in <i>All of Us</i> to my work
+        <IndentedUnorderedList>
+          <li>I will acknowledge the <AoUTitle/> and its research participants
+            in all oral and written presentations, disclosures, and publications resulting from
+            any analyses of the data.
+            <IndentedUnorderedList>
+              <li>Here is an example acknowledgement statement:</li>
+              <IndentedUnorderedList>
+                <IndentedListItem>
+                  “The <AoUTitle/> is supported by the National Institutes of Health, Office of the
+                  Director: Regional Medical Centers: 1 OT2 OD026549; 1 OT2 OD026554; 1 OT2 OD026557; 1 OT2 OD026556;
+                  1 OT2 OD026550; 1 OT2 OD 026552; 1 OT2 OD026553; 1 OT2 OD026548; 1 OT2 OD026551; 1 OT2 OD026555;
+                  IAA #: AOD 16037; Federally Qualified Health Centers: HHSN 263201600085U; Data and Research Center:
+                  5 U2C OD023196; Biobank: 1 U24 OD023121; The Participant Center: U24 OD023176; Participant Technology
+                  Systems Center: 1 U24 OD023163; Communications and Engagement: 3 OT2 OD023205; 3 OT2 OD023206;
+                  and Community Partners: 1 OT2 OD025277; 3 OT2 OD025315; 1 OT2 OD025337; 1 OT2 OD025276. In addition,
+                  the <AoUTitle/> would not be possible without the partnership of its participants.”
+                </IndentedListItem>
+                <IndentedListItem>
+                  I will submit an electronic version of a final, peer-reviewed manuscript to PubMed Central
+                  immediately upon acceptance for publication, to be made publicly available no later than 12
+                  months after the official date of publication.
+                </IndentedListItem>
+              </IndentedUnorderedList>
+            </IndentedUnorderedList>
+          </li>
+        </IndentedUnorderedList>
+      </li>
+    </IndentedUnorderedList>
+    <SecondHeader>As “Authorized Demonstration User” of the <AoUTitle/> data, I
+      will:</SecondHeader>
+    <IndentedUnorderedList>
+      <li><strong>NOT</strong> share my login information with anyone.
+        <IndentedUnorderedList>
+          <li>I will not share my login information with another authorized demonstration or other user
+            of the <AoUTitle/>.
+          </li>
+          <li>I will not create any group or shared accounts.</li>
+        </IndentedUnorderedList>
+      </li>
+      <li>
+        <strong>NOT</strong> use <AoUTitle/> data or any external data, files, or software that I upload
+        into the Research Workbench for research that is discriminatory or stigmatizing of individuals, families,
+        groups, or communities. Please review the All of Us policy on stigmatizing research here.
```

Comment:
`here` appears to link to a doc. We have a stigmatization page, maybe its the same text? (What opens when you click the stigmatization link from create workspace.)

Reply:
bump on the link needed here. https://docs.google.com/document/d/1RGWCmujvunLRdlBTYpZArcben4QNFwOhp7mSDK3puC0/edit